secant/UIComponents/Buttons/NeumorphicDesignModifier.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  NeumorphicDesignModifier.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 02/21/22.
+//
+
+import SwiftUI
+
+/// Neumorphic design is charasterictical with two shadows (light & dark) around the view
+/// Appereance in our case is influenced by two parameters:
+/// - Parameters:
+///   - colorScheme: The light is using full neumorphic design while dark is limited to soft shadow only
+///   - isPressed: When the button is pressed, there are different behaviours for light vs. dark colorScheme
+/// This design is mostly used for CircularFrame, not designed for a button (see NeumorphicButtonDesign)
+// swiftlint:disable:next private_over_fileprivate strict_fileprivate
+fileprivate struct Neumorphic: ViewModifier {
+    @Environment(\.colorScheme) var colorScheme
+
+    let isPressed: Bool
+    
+    init(_ isPressed: Bool) {
+        self.isPressed = isPressed
+    }
+    
+    func body(content: Content) -> some View {
+        content
+            .shadow(
+                color: Asset.Colors.Onboarding.neumorphicDarkSide.color,
+                radius: 15,
+                x: colorScheme == .light && !isPressed ? 10 : -10,
+                y: colorScheme == .light && !isPressed ? 10 : 10
+            )
+            .shadow(
+                color: Asset.Colors.Onboarding.neumorphicLightSide.color,
+                radius: 10,
+                x: colorScheme == .light && !isPressed ? -12 : 12,
+                y: colorScheme == .light && !isPressed ? -12 : -12
```

Comment:
same as above

Reply:
good catch, it was different but after latest updates I missed the equality

Reply:
Happens to me all the time ^^